client/src/ui/molecules/breadcrumbs/index.tsx

```diff
@@ -9,7 +9,7 @@ export function Breadcrumbs({ parents }: { parents: DocParent[] }) {
   }
 
   return (
-    <nav className="breadcrumbs-container">
+    <nav className="breadcrumbs-container" aria-label="Breadcrumb navigation">
```

Comment:
Alternatively a `<span>` that is visible for screenreaders only?

Reply:
In this case, it is mainly set to uniquely identify this landmark to screen reader users. We could also add a visually hidden heading element but, I believe we would still need to connect the header and the landmark with `aria-describedby` so, I reckon this is sufficient for the purpose. Thanks, @Ryuno-Ki